lib/chef/provider/powershell_script.rb

```diff
@@ -29,20 +29,6 @@ def initialize (new_resource, run_context)
         add_exit_status_wrapper
       end
 
-      def action_run
-        validate_script_syntax!
-        super
-      end
-
-      def command
```

Comment:
Why did you remote command?  I don't think windows_script sets the interpreter path correctly.